tests/suites/test_suite_mpi.function

```diff
@@ -768,43 +768,46 @@ void mpi_core_cond_assign( data_t * input_X,
                            data_t * input_Y )
 {
     #define MAX_LEN 64
```

Comment:
I don't expect `MAX_LEN` to be used in any of the tests. There's no reason for the magic constant 64 to appear.